boa_engine/src/bigint.rs

```diff
@@ -407,7 +407,22 @@ impl From<usize> for JsBigInt {
         }
     }
 }
-
+impl From<i128> for JsBigInt {
+    #[inline]
+    fn from(value: i128) -> Self {
+        Self {
+            inner: Rc::new(RawBigInt::from(value)),
+        }
+    }
+}
+impl From<u128> for JsBigInt {
+    #[inline]
+    fn from(value: u128) -> Self {
+        Self {
+            inner: Rc::new(RawBigInt::from(value)),
+        }
+    }
+}
```

Comment:
This is fine (conversion from `u/i128` to `JsBigInt`), but I don't think we should add for `Numeric`.

Reply:
Okay I can make that change

Reply:
Resolved by the latest commit to the PR, thanks for the feedback @HalidOdat

Reply:
I meant that we should keep the `u/i128` to `JsBigInt` conversion, and only remove the `u/i128` to `Numeric` conversion.

Reply:
Oh sorry. But the `JsBigInt` conversion requires `Numeric` trait to be implemented

Reply:
Sorry, that was my mistake I had it backward, I re-added the trait impl for `JsBigInt`